core/src/components/datetime/datetime.ionic.scss

```diff
@@ -216,6 +216,13 @@
 
 // Calendar / Footer / Action Buttons
 // -----------------------------------
+:host .datetime-footer {
+  @include globals.padding(globals.$ion-space-200, globals.$ion-space-0, globals.$ion-space-200, globals.$ion-space-0);
+  margin-top: globals.$ion-space-200;
+
+  border-top: globals.$ion-border-style-solid globals.$ion-border-size-025 globals.$ion-primitives-neutral-300;
```

Comment:
Where are these design details being pulled from? From the Figma, there is no border on the datepicker, unless it's been updated in a place I'm not aware of?

Reply:
Awesome! Looks like you also need to update your screenshots